Cargo.toml

```diff
@@ -49,6 +49,7 @@ windows = { version = "0.43", features = [
     "Win32_Media_Multimedia",
     "Win32_Media_Audio",
 ] }
+parking_lot = { version = "0.12.1" }
```

Comment:
Could this be made conditional only on Windows?

Reply:
That's what I did. Line 45 is specifying the target as windows-only.